ims/database/db_connection.py

```diff
@@ -15,18 +19,15 @@ class DatabaseConnection:
     # sample_bmi.db should be changed to something more realistic
     # NullPool pool class is equivalent to no connection pool
     # Should be adapted to postgres SQL
-    engine = create_engine('sqlite:///sample_bmi.db', poolclass=NullPool)
+    engine = create_engine('sqlite:///' + _cfg.db_url, poolclass=NullPool)
 
```

Comment:
Will this be sqlite again, even if we change it to postgres? Shouldn't we change it to something generic in properties file to dynamically create a URL based on backend chosen?